lib/commands/favorite.go

```diff
@@ -43,12 +71,29 @@ func (c *Cmd) ListFavorites(cCtx *cli.Context) error {
 // argument it is used, otherwise the user is walked through a wizard to make a
 // selection.
 func (c *Cmd) Favorites(cCtx *cli.Context) error {
-	// map our favorites for ease of use
-	fNames, fMap := helper.MapFavs(c.config.Favorites)
+
+	// get the combined list of favorites from the CLI config and the Kion API (if compatible)
```

Comment:
Can we dry this up? This flow through line 90 is a duplicate of lines 20-35 above. Pull it out into a local func in this same package and call that.